docs/control-structures.rst

```diff
@@ -171,6 +194,9 @@ throws an exception or goes out of gas.
     external functions happen after any changes to state variables in your contract
     so your contract is not vulnerable to a reentrancy exploit.
 
+...note::
+    A function call from one contract to another does not create its own transaction, it is a message call as part of the overall transaction. This is also why several block explorer do not show Ether sent between contracts correctly.
```

Comment:
"several block explorer**s**"

Reply:
Also this came from the FAQ. I think we could remove the second sentence or at least reword it:
> Block explorers may not show calls and as a result Ether sent during transactions properly.

Reply:
I just removed it.